src/app/utils/masking.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Converts a number to a comma separated value
+ * @param {Number} value
+ * @returns {string}
+ */
+export function convertToThousandsSeparatedString(value: number): string {
+  let maskValue = "";
+  maskValue = Math.round(value).toLocaleString();
+
+  return maskValue;
+}
+
+/**
+ * Converts a float number to a percentage string
+ * @param {Number} value
+ * @returns {string}
+ */
+export function convertPercentage(value: number): string {
+  let maskValue = value * 100;
+  maskValue = Math.round(maskValue);
+
+  return maskValue.toString();
+}
```

Comment:
if we're converting to a string is it worth adding the percentage sign % here as well?

Reply:
I thought about it, same with the dollar sign, but I felt like that muddied the purpose of the function